README.md

```diff
@@ -40,22 +45,19 @@ et al, which specialise in preparing your spatial data for visualisation. Rather
 choice of tool to you the user, as we want GeoVista to remain as flexible and open-ended as possible to the entire
 Scientific Python community.
 
-Simply put, "*GeoVista is to PyVista*", as "*Cartopy is to Matplotlib*". Well, that's the aspiration.
+Simply put, "*GeoVista is to PyVista*", as "*[Cartopy](https://scitools.org.uk/cartopy/docs/latest/) is to
```

Comment:
Nice :+1: 

Why not also add the doc links for both `GeoVista` and `PyVista` too.

Reply:
Yes, personal taste this one. There are already links elsewhere for these but perhaps consistency is important 👍

Reply:
1abe1eb6d0a4ef9b8b26d6f3df401d8631186ac0